### PR TITLE
JAVA-1810: Note which setters are not propagated to PreparedStatement

### PR DESCRIPTION
**Pull request by tolbertam**
Just a minor doc change for [JAVA-1810](https://datastax-oss.atlassian.net/browse/JAVA-1810).

### CODE REVIEW

File: driver-core/src/main/java/com/datastax/driver/core/Statement.java [hunk 1]
```diff
@@ -224,6 +224,9 @@ public abstract ByteBuffer getRoutingKey(
    * method is thus only useful in case you want to punctually override the default policy for this
    * request.
    *
+   * <p>Note that unlike other configuration, when this statement is prepared {@link
+   * BoundStatement}s created off of {@link PreparedStatement} do not inherit this configuration.
+   *
```
Comment (olim7t) [inline driver-core/src/main/java/com/datastax/driver/core/Statement.java]: It looks like the retry policy is actually propagated: https://github.com/datastax/java-driver/blob/3.x/driver-core/src/main/java/com/datastax/driver/core/AbstractSession.java#L131

**Review by olim7t — CHANGES_REQUESTED** [review]

File: driver-core/src/main/java/com/datastax/driver/core/PreparedStatement.java [hunk 2]
```diff
@@ -125,7 +125,7 @@
    * Sets a default consistency level for all bound statements created from this prepared statement.
    *
    * <p>If no consistency level is set through this method, the bound statement created from this
-   * object will use the default consistency level (ONE).
+   * object will use the default consistency level (LOCAL_ONE).
```
Comment (olim7t) [inline driver-core/src/main/java/com/datastax/driver/core/PreparedStatement.java:128]: Not related to this PR, but it was wrong. We'll keep the commit separate.

File: driver-core/src/main/java/com/datastax/driver/core/Session.java [hunk 3]
```diff
@@ -249,9 +245,14 @@
   /**
    * Prepares the provided query.
    *
-   * <p>This method behaves like {@link #prepare(String)}, but note that the resulting {@code
-   * PreparedStatement} will inherit the query properties set on {@code statement}. Concretely, this
-   * means that in the following code:
+   * <p>This method behaves like {@link #prepare(String)}, but the resulting {@code
```
Comment (tolbertam) [inline driver-core/src/main/java/com/datastax/driver/core/Session.java:252]: Nice :+1:

File: driver-core/src/main/java/com/datastax/driver/core/Session.java [hunk 4]
```diff
@@ -15,11 +15,7 @@
  */
 package com.datastax.driver.core;
 
-import com.datastax.driver.core.exceptions.AuthenticationException;
-import com.datastax.driver.core.exceptions.NoHostAvailableException;
-import com.datastax.driver.core.exceptions.QueryExecutionException;
-import com.datastax.driver.core.exceptions.QueryValidationException;
-import com.datastax.driver.core.exceptions.UnsupportedFeatureException;
+import com.datastax.driver.core.exceptions.*;
```
Comment (tolbertam) [inline driver-core/src/main/java/com/datastax/driver/core/Session.java]: Small nit, google formatter doesn't catch this, but we should not be using wildcard imports.

File: driver-core/src/main/java/com/datastax/driver/core/Statement.java (hunk 1 above)
Comment (tolbertam) [reply driver-core/src/main/java/com/datastax/driver/core/Statement.java]: Ah, I missed that somehow, good catch!

File: driver-core/src/main/java/com/datastax/driver/core/PreparedStatement.java (hunk 2 above)
Comment (tolbertam) [reply driver-core/src/main/java/com/datastax/driver/core/PreparedStatement.java:128]: Good catch

**Review by olim7t — APPROVED** [review]

**Comment by tolbertam** [comment]
Thanks for reviewing and improving the docs on the prepare method, merging :+1:

**Pull request merged**